README.md

```diff
@@ -1 +1,156 @@
-# To-do-list
\ No newline at end of file
+# AWESOME BOOK PROJECT
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [AWESOME BOOK PROJECT] <a name="about-project"></a>
+
+**[AWESOME_BOOK_PROJECT]** 
+This project is about addition and deletion of books into local storage 
+using javascript objects. 
```

Comment:
- [ ] While this project is for the TODO APP, your README file describes the Awesome book project. Could you fix this to avoid any problems? Always take the time to make your READMEs as descriptive as possible using the Microverse [README template](https://github.com/microverseinc/readme-template) as a guide.